content/millennium/r4/financial/financial-transaction.md

```diff
@@ -39,7 +39,7 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  `financial-transaction-allocation`      | None (contains nested extensions) | Defines how the payment or adjustment is to be allocated across other resources.
  `financial-transaction-amount`          | [`Money`]                         | The total amount of the financial transaction.
  `financial-transaction-card-brand`      | [`string`]                        | Identifies the brand of credit card when credit card is used as a payment method.
- `financial-transaction-date`            | [`date`]                          | Represents the expiration date if method is card, check date if method is check, EFT date if method is EFT or Lockbox date if method is lockbox.
+ `financial-transaction-date`            | [`date`]                          | Represents the expiration date if method is card, check date if method is check.
```

Comment:
Can this read 'Represents the expiration date if method is card or check date if method is check.'

Reply:
updated here.
https://github.com/cerner/fhir.cerner.com/pull/455/commits/a19a9a85fb67aaadc8eb7d2893b02d8363b09316#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR42